app/src/main/res/values/strings_questionnaire.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="questionnaire_title_age">Age</string>
+    <string name="questionnaire_description_age">How old are you?</string>
+    <string-array name="questionnaire_keys_age">
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_keys_age" is not translated in "ja" (Japanese) [MissingTranslation]